docs/swagger.json

```diff
@@ -0,0 +1,651 @@
+				],
+				"summary": "Saves a contract",
+				"description": "",
+				"operationId": "saveContract",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Contract"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		},
+		"/documents": {
+			"post": {
+				"tags": [
+					"Corellia"
+				],
+				"summary": "Saves a document",
+				"description": "",
+				"operationId": "saveDocument",
+				"produces": [
+					"application/json"
+				],
+				"parameters": [
+					{
+						"in": "body",
+						"name": "body",
+						"required": false,
+						"schema": {
+							"$ref": "#/definitions/Document"
+						}
+					}
+				],
+				"responses": {
+					"200": {
+						"description": "successful operation",
+						"schema": {
+							"type": "string"
+						}
+					}
+				}
+			}
+		}
+	},
+	"definitions": {
+		"Address": {
+			"type": "object",
+			"required": [
+				"city",
+				"houseNumber",
+				"street",
+				"zipCode"
+			],
+			"properties": {
+				"street": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"houseNumber": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"zipCode": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"city": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				}
+			}
+		},
+		"Agent": {
+			"type": "object",
+			"required": [
+				"agentNumber",
+				"email"
+			],
+			"properties": {
+				"agentNumber": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"email": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				}
+			}
+		},
+		"Company": {
+			"type": "object",
+			"required": [
+				"address",
+				"legalForm",
+				"name",
+				"uidNumber"
+			],
+			"properties": {
+				"name": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"legalForm": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"uidNumber": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"address": {
+					"$ref": "#/definitions/Address"
+				}
+			}
+		},
+		"CompanyInvoiceAddress": {
+			"type": "object",
+			"required": [
+				"address",
+				"name"
+			],
+			"properties": {
+				"name": {
+					"type": "string",
+					"minLength": 0,
+					"maxLength": 100
+				},
+				"address": {
+					"$ref": "#/definitions/Address"
+				}
+			}
+		},
+		"Contract": {
+			"type": "object",
+			"required": [
+				"agent",
+				"companyInvoiceAddress",
+				"discountCode",
+				"distributor",
+				"endDate",
+				"id",
+				"personInvoiceAddress",
+				"policyId",
+				"policyIssueDate",
+				"product",
```

Comment:
There are a lot of fields here?  Perhaps you could provide the CSV we send you, with each field here referenced, so we have mapping table?

Reply:
accepted, we will provide you with a mapping table next week

Reply:
around 20.03.